src/Test/API/AbstractPluginActionsTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace CF\API\Test;
+
+class AbstractPluginActionsTest extends \PHPUnit_Framework_TestCase
```

Comment:
Since "cloudflare_plugin_backend" code should be really stable we should have the test coverage really high. Everything including getters, setters to PluginRoutes, PluginSettingsKeys to endpoints like patchPluginDefaultSettings should all be covered.

I didn't write comments on the functions one by one since there were too many.